impeller/renderer/backend/vulkan/render_pass_vk.cc

```diff
@@ -263,6 +282,10 @@ SharedHandleVK<vk::Framebuffer> RenderPassVK::CreateVKFramebuffer(
 
   fb_info.setAttachments(attachments);
 
+  if (old_framebuffer != nullptr) {
+    return old_framebuffer;
+  }
```

Comment:
In this case, it looks like the work above here is not necessary if we have an old_framebuffer.  It seems to just be building the `attachments` variable which is thrown away.  It would be more clear if here we could also say `frame_buffer = recycled_frame_buffer ? recycled_frame_buffer : CreateVKFramebuffer(context, pass)`.

Reply:
Done